DQM/SiPixelPhase1Common/src/SiPixelCoordinates.cc

```diff
@@ -0,0 +1,916 @@
+// -*- C++ -*-
```

Comment:
@jkarancs looked at it, but I don't see any obvious candidate... 
@ianna Wow, thanks! I guess this was loaded as a dependency of sth. in the DQM package.

Now just lots of linking errors, but there it is more clear what needs to be added...